src/ucp/api/ucp.h

```diff
@@ -1295,6 +1308,77 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh);
 
 /**
  * @ingroup UCP_MEM
+ * @brief list of UCP memory use advice.
+ *
+ */
+typedef enum ucp_mem_advice {
+    UCP_MADV_NORMAL   = 0,  /**< No special treatment */
+    UCP_MADV_WILLNEED,      /**< This advice be used on the memory that was 
+                                 mapped with the @ref UCP_MEM_MAP_NONBLOCK 
+                                 flag in order to speed up memory mapping and
```

Comment:
Please change to read more like the UCT_MADV_WILLNEED description, e.g.,
  can be used on the memory mapped with
  @ref UCP_MEM_MAP_NONBLOCK to speed up
  memory mapping and to avoid page faults when
  the memory is accessed for the first time. */